test/system/helpers.bash

```diff
@@ -341,6 +341,16 @@ function is_cgroupsv2() {
     test "$cgroup_type" = "cgroup2fs"
 }
 
+# True if podman is using netavark
+function is_netavark() {
+    local be_type
+    be_type=$($PODMAN info --format '{{.Host.NetworkBackend}}')
```

Comment:
Is the use of `$PODMAN` here deliberate? (As opposed to `run_podman` and using `$output`)? The only important reason to use `$PODMAN` is when it's ultra-important to use the local client version instead of `-remote`, and I don't think in this case it makes a difference. This whole function could be two lines:
```
run_podman info --format ...
test "$output" =~ "netavark"
```

Reply:
Yes it was on purpose.  I didn't want any chance of secondary issues conflating the (simple) query with complexities of a complex function.  It's also much faster this way, and seemed to be the pattern of all the other `is_XYZ()` functions, so I went with it.  I'll change it, but it's only out of expediency getting this PR merged.